docs/source/tuning.rst

```diff
@@ -16,35 +16,13 @@
 .. cartographer_ros SHA: 99c23b6ac7874f7974e9ed808ace841da6f2c8b0
 .. TODO(hrapp): mention insert_free_space somewhere
 
-Tuning
-======
+Tuning methodology
+==================
 
 Tuning Cartographer is unfortunately really difficult.
 The system has many parameters many of which affect each other.
 This tuning guide tries to explain a principled approach on concrete examples.
 
-Two systems
```

Comment:
I'm not sure, why delete these parts?

Reply:
The main idea of my patchset was actually to replace this short paragraph with a much more comprehensive "algo_walkthrough.rst". I notice you had no comments on that file, I assume you might have missed it because it is by far the one that requires the most careful review. The diff is so big that GitHub minimizes the content of the file and you don't see it by default.